src/styles/variables.scss

```diff
@@ -1,15 +1,21 @@
 :root {
-  --color-white: #f1f2f9;
-  --color-accent: #905bff;
-  --color-green: #27ae60;
-  --color-red: #eb5757;
+  --primeTextColor: #f1f2f9;
+  --secondaryTextColor: #75767f;
 
-  --color-black: #0f1121;
-  --color-surface-one: #161827;
-  --color-surface-two: #323542;
-  --color-elements: #3b3e4a;
-  --color-icons: #4a4d58;
-  --color-secondary: #75767f;
+  --primeAccentColor: #905bff;
+  --secondaryAccentColor: #a378ff;
+  --selectedFavoriteColor: #eb5757;
+
+  --primeBgColor: #0f1121;
+  --secondaryBGColor: #161827;
+  --thirdaryBgColor: #323542;
+  --elementsColor: #3b3e4a;
+  --iconsColor: #4a4d58;
```

Comment:
im not sure if those names are descriptive enough? especially "thirdary" looks super weird
someone else from the team should look and give their opinion